src/app/shared/model/Terminal.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Note:
+// Payload format for bidirectional WebSocket(WS) communication
+export interface ITerminalFrame {
+  Op: string;
+  Data?: string;
+  SessionID?: string;
+  Rows?: number;
+  Cols?: number;
```

Comment:
Is it coming from some API or is it internal? If internal please follow our naming conventions.

Reply:
This API format
https://github.com/kubermatic/kubermatic/blob/master/pkg/handler/websocket/terminal.go#L63